express-intro/server.ts

```diff
@@ -0,0 +1,13 @@
+import express from 'express';
+
+const app = express();
+
+app.listen(8080, () => {
+  console.log('Express server listening on port 8080');
+});
+
+app.use((req, res, next) => {
+  console.log('Hello, World!');
+  console.log('The date is', new Date());
+  next();
+});
```

Comment:
Hey Connie, just make sure to place this code before the `app.listen` call

app.listen, should be the last app method in the file when you build a server

Reply:
got it! fixed 
![Screenshot 2024-04-04 at 11 55 45 AM](https://github.com/connie-chin/rc0124-code-solutions/assets/155609517/1358710e-cba6-4e5a-b647-9afda4c22f80)